content/kotsadm/installing/airgap-packages.md

```diff
@@ -8,6 +8,34 @@ draft: false
 
 When installing an application from an airgap package, the container images and application manifests are provided by the application vendor in an archive that can be used to deliver the artifacts into the cluster. This feature is only available for licenses that have the airgap feature enabled.
 
+## Kots install
+Begin by [installing the Kots CLI kubectl plugin](/kots-cli/getting-started/).  The Admin Console can be installed using the CLI and the airgap package included in the [release assets](https://github.com/replicatedhq/kots/releases) named `kotsadm.tar.gz`.   The asset version has to match the CLI version, which can be determined by running:
+
+```shell
+kubectl kots version
+```
+
+First step is to extract kotsadm images and push them into a private registry.  Registry credentials provided in this step need to have push access.  These credentials will not be stored anywhere.
+
+```shell
+kubectl kots admin-console push-images ./kotsadm.tar.gz private.registry.host/kotsadm --registry-username rw-user --registry-password rw-password
+```
+
+The next step is to install the Admin Console using images pushed in the previous step.  Registry credentials provided in this step only need to have read access, and they will be stored in a kubernetes secret in the same namespace where Admin Console will be installed.
```

Comment:
stored in a Kubernetes secret in the same namespace where Admin Console will be installed.
These credentials will be used to pull the images, and will be automatically created as an imagePullSecret on all of the Admin Console pods.